extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/Args.scala

```diff
@@ -1,10 +1,10 @@
 package org.broadinstitute.monster.encode.extraction
 
 import caseapp.{AppName, AppVersion, HelpMessage, ProgName}
-import org.broadinstitute.monster.EncodeIngestBuildInfo
```

Comment:
I like that you renamed this to be extraction specific since "Ingest" could be confusing.
We're only adding transformation and orchestration pieces, right?

Reply:
yeah I'm following the format Dan made for the dog-aging-ingest, which has extraction and transformation. One of our plugins generates the BuildInfo thing so it updates the name based on the project 🙂 very useful